src/media/css/search.styl

```diff
@@ -1,6 +1,8 @@
 @import 'lib';
 
 .no-results {
+    font-weight: 500;
```

Comment:
Oh I agree...it was requested that it's 100% the same as before (I kind of ignored the padding) thus the weirdo old color also.